### DISCUSSION TITLE
Organiser les catégories

### DISCUSSION THREAD

**Discussion by detobel36** [Kodi-vStream/venom-xbmc-addons#3444] [Idées]
Salut,

Je propose d'organiser les différentes catégories notées ici :smiley:     

Je propose de:
- garder "Announcements" mais de le renommer en "Annonces"
- de renomer "Q&A" en "support" ou "demande d'aide"
- de renomer "ideas" en "Idée"
- de supprimer "show and tell"

**Comment by sizanic** [comment]
Salut,
Ce que tu proposes, c'est de traduire les catégories en français 😄 

Bonne idée 👍🏻 
Dès que je trouve comment on fait 🤔

**Comment by sizanic** [comment]
C'est fait
